spec/active_record/connection_adapters/oracle_enhanced_data_types_spec.rb

```diff
@@ -882,6 +882,234 @@ class ::TestSerializeEmployee < ActiveRecord::Base
   end
 end
 
+describe "OracleEnhancedAdapter handling of NCLOB columns" do
+  include SchemaSpecHelper
+
+  before(:all) do
+    ActiveRecord::Base.establish_connection(CONNECTION_PARAMS)
+    @conn = ActiveRecord::Base.connection
+    schema_define do
+      create_table :test_employees, force: true do |t|
+        t.string  :first_name, limit: 20
+        t.string  :last_name, limit: 25
+        t.column  :comments, :ntext
```

Comment:
I think `t.ntext   :comments` looks better for migrations.  you can refer the commit below to support `t.ntext` for migrations.

https://github.com/yahonda/oracle-enhanced/commit/7ee3bb367d3ce450adb8ae136ee59b0624cac40d